### ISSUE TITLE
CDF, etc

### ISSUE THREAD

**Issue by cscherrer** [oschulz/DistributionMeasures.jl#5]
Ideally, we could define functions like `cdf` in MeasureBase. Unfortunately, these names are "owned" by Distributions.jl. Should this package define, for example,
```julia
Distributions.cdf(::MeasureBase.StdNormal, z) = StatsFuns.normcdf(z)
```
?

This would be "licensed" type piracy, but I don't see a better option.

**Comment by oschulz** [comment]
For measures maybe we should aim for something that works on "interval-like" arguments instead? Then we'd want a different name anyway. :-)

**Comment by cscherrer** [comment]
Sure, but `cdf` is more of a primitive. It really should be in something like `StatsBase`, but that's another discussion.

For "interval-like arguments", I think we just have application. Mathematically it's $\mu([a,b])$, for us probably `μ(a..b)`

**Comment by oschulz** [comment]
Yes, I think `IntervalSets.Interval` could be a good start. Obviously something more complex will be needed for higher dims (but maybe vectors of intervals for now)?

**Comment by cscherrer** [comment]
I think https://github.com/cscherrer/MeasureBase.jl/pull/88 has this covered. Your ideas of an `IntervalSets.Interval` basis and the transport API work out really nicely.
